app.go

```diff
@@ -197,6 +210,16 @@ func (a *App) RunAsSubcommand(ctx *Context) error {
 		}
 	}
 
+	if a.After != nil {
+		defer func() {
+			aferr := a.After(context)
+			// check Action error to avoid shadowing
+			if err == nil {
```

Comment:
Done! It builds and passes tests, so it looks good. If you find anything to change, don't hesitate to contact me back.